geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/ListenCommand.scala

```diff
@@ -89,7 +90,7 @@ class ListenMessageFormatter extends MessageFormatter {
     decoder = new KafkaGeoMessageDecoder(sft)
   }
 
-  override def writeTo(key: Array[Byte], value: Array[Byte], output: PrintStream): Unit = {
+  def writeTo(key: Array[Byte], value: Array[Byte], output: PrintStream): Unit = {
```

Comment:
This was more of a band-aid solution to get the tools module to not complain when building. It had depended on geomesa-kafka-utils-common.